src/main/java/com/rakovets/course/java/core/practice/decision_making_statements/Task03.java

```diff
@@ -37,6 +37,17 @@ public static void main(String[] args) {
     static String getColorHealthPoint(int currentHealthPoint, int maxHealthPoint) {
         //TODO
         // Код, решающий задачу пишем ниже, при этом используя параметры метода
-        return null;
+        double healthPointInPercent = currentHealthPoint * 100.0/ maxHealthPoint;
+        if (healthPointInPercent > 0.0 && healthPointInPercent < 25.0) {
+            return "RED";
+        } else if (healthPointInPercent >= 25.0 && healthPointInPercent < 50.0) {
+            return "ORANGE";
+        } else if (healthPointInPercent >= 50.0 && healthPointInPercent < 75.0) {
+            return "YELLOW";
+        } else if (healthPointInPercent >=75.0 && healthPointInPercent <= 100.0) {
```

Comment:
"Violation: There isn't separating white spaces for operators OR parameters
Solution: Add white spaces"